contrib/depends/packages/ldns.mk

```diff
@@ -1,8 +1,8 @@
 package=ldns
-$(package)_version=1.6.17
+$(package)_version=1.7.1
```

Comment:
This patch also looks different from the original. If it's your original, please make a PR on Monero first.

If your cherry-picking results in a lot of merge conflicts and you have to modify the original patch significantly, then consider redoing it in the original order.